common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -341,6 +341,9 @@ siege_works_02 = {
 	character_modifier = {
 		siege_weapon_siege_value_mult = 0.3
```

Comment:
Your arts are of different size.
Big arts must be 680x400.
Small arts must be 160x160.

I would bring back vanilla arts for vanilla units. I like them more.

I would use [this art](https://wow.gamepedia.com/File:Troll_Wars_magi.jpg) for the casters. First, it's not a gnome. Second, it's magic in action.

I would use these arts for the light aerial. It shows air combat.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103772640-45434700-5043-11eb-9b15-9fbdee8b095c.png)
![image](https://user-images.githubusercontent.com/46384992/103772714-660b9c80-5043-11eb-984e-b6db05e276d4.png)

</details>

If you use paint.net, use these settings.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103772028-3e680480-5042-11eb-8cc7-2a0231e84eb2.png)

</details>

I would use these arts for the massive aerial.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103772080-593a7900-5042-11eb-93a4-7bb95b8d56b0.png)
![image](https://user-images.githubusercontent.com/46384992/103772235-9d2d7e00-5042-11eb-88f6-2dc2df1fa7cd.png)

</details>